regression/rt_gk_wham_1x2v_p1.c

```diff
@@ -865,6 +865,7 @@ int main(int argc, char **argv)
     .polarization_bmag = ctx.B_p, 
     .fem_parbc = GKYL_FEM_PARPROJ_NONE,
     .kperpSq = pow(ctx.kperp, 2.),
+    .is_static = true,
```

Comment:
i don't think this belongs here. In general we don't wish to run with static fields, it is only a dev tool, and we want reg tests to test production functionality